src/Illuminate/Routing/Middleware/ThrottleRequests.php

```diff
@@ -287,8 +287,8 @@ protected function getHeaders($maxAttempts,
         ];
 
         if (! is_null($retryAfter)) {
-            $headers['Retry-After'] = $retryAfter;
-            $headers['X-RateLimit-Reset'] = $this->availableAt($retryAfter);
+            $headers['Retry-After'] = $retryAfter + 1;
+            $headers['X-RateLimit-Reset'] = $this->availableAt($retryAfter + 1);
```

Comment:
Changes to `Retry-After` looks logical but not sure about `X-RateLimit-Reset`

Reply:
> The GitHub API rate limit ensures that the API is fast and available for everyone. If you hit a rate limit, you should stop making requests **until after the time specified** by the x-ratelimit-reset header.

Reply:
@crynobone the problem I see with that is we are showing a lie. You can't make requests after that moment in time - you have to wait until of the second.

The rate limit doesn't reset at the value in `X-RateLimit-Reset`. It resets at the value in `X-RateLimit-Reset` + 999 milliseconds.

I dunno. I struggle with this. I can see this one both ways.